CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ Security - in case of vulnerabilities.
 
 ## [Unreleased]
 
+## [2.4.1] 2019-07-18
```

Comment:
Not necessarily, we do not release this library for each change. According symver you can add change log under `[Unreleased]`.

Reply:
I agree, we can add change log under `Unreleased`